examples/append_address_to_webhook.rs

```diff
@@ -0,0 +1,15 @@
+use helius::types::Cluster;
+use helius::Helius;
+
+#[tokio::main]
+async fn main() {
```

Comment:
This should have a return type. You could import `Result` from `use helius::error::Result` and use the type alias for the return type so the line reads `async fn main() -> Result<()> {`. You'll also have to add in `Ok(())` at the end of the block to satisfy the return type. If you get confused, I recommend looking at the other examples we currently have for the SDK